src/loaders/OmeZarrLoader.ts

```diff
@@ -483,10 +520,17 @@ class OMEZarrLoader implements IVolumeLoader {
         }
       });
 
-      const result = (await level.getRaw(sliceSpec)) as RawArray;
-      const u8 = convertChannel(result.data, result.dtype);
-      vol.setChannelDataFromVolume(ch, u8);
-      onChannelLoaded?.(vol, ch);
+      try {
+        const result = (await level.getRaw(sliceSpec, { concurrencyLimit: Infinity })) as RawArray;
+        const u8 = convertChannel(result.data, result.dtype);
+        vol.setChannelDataFromVolume(ch, u8);
+        onChannelLoaded?.(vol, ch);
+      } catch (e) {
+        // TODO: verify that cancelling requests in progress doesn't leak memory
+        if (e !== CHUNK_REQUEST_CANCEL_REASON) {
+          throw e;
+        }
+      }
```

Comment:
When/how will you verify this?

Reply:
My plan is to verify this as part of work on #154, which will add the new layer of per-subscriber `Promise` wrapping we talked about earlier. The issue here has to do with whether or not cancelling requests means all associated promises are either rejected (easy to verify) or lose all references and are garbage collected (hard to verify). I believe we can get close to certain on this with the devtools profiler, though it might be helpful to remedy our other known memory leak (#143/#156) first.